toolkits/global/packages/global-autocomplete/js/index.js

```diff
@@ -11,8 +11,11 @@ const autoComplete = arguments_ => {
 		inputDelay = 300,
 		requestTimeout = 2000,
 		headers = {},
+		httpMethod = 'GET',
+		bodyTemplate,
```

Comment:
This is cool, could you maybe add it to the demo stuff so that it is documented, otherwise it might not be clear

Reply:
I will, very good point. I though about the tests but not the demo 🤦‍♂️